Classes/Public/Manager/Objects/OCTMessageCall.h

```diff
@@ -6,19 +6,19 @@
 //  Copyright (c) 2015 dvor. All rights reserved.
 //
 
-#import "OCTMessageAbstract.h"
+#import "OCTObject.h"
 #import "OCTManagerConstants.h"
 
-@interface OCTMessageCall : OCTMessageAbstract
+@interface OCTMessageCall : OCTObject
 
 /**
  * The length of the call in seconds.
  **/
-@property (assign, nonatomic, readonly) NSTimeInterval callDuration;
+@property  NSTimeInterval callDuration;
```

Comment:
Also `messageCall.callEvent` and `call.state` seems to be quite similar. Maybe we can remove can remove `callEvent` from message and just use `state` of the call?